.eslintrc.js

```diff
@@ -5,9 +5,14 @@ module.exports = {
   parser: 'babel-eslint',
   extends: 'airbnb-base',
   env: {
+    'angular/mocks': true,
     browser: true,
     jquery: true,
   },
+  plugins: [
+    'angular',
```

Comment:
Maybe we could enable some rules by the way 😄 

https://www.npmjs.com/package/eslint-plugin-angular#rules

Reply:
@FredericEspiau Sure!

Is there any rules that you want to be turned on?

If you have some, let's discuss about it;)